### PR TITLE
adding Gaboardi and Amaral as maintainers

### PR DESCRIPTION
**Pull request by jGaboardi**
This PR addresses #7 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,4 +117,6 @@ Feedstock Maintainers
 =====================
 
 * [@ocefpaf](https://github.com/ocefpaf/)
+* [@pedrovma](https://github.com/pedrovma)
+* [@jGaboardi](https://github.com/jGaboardi)
```
Comment (ocefpaf) [inline README.md]: The README is Auto generated, sorry for not mentioning this before. You need to add your GH handles in the meta.yaml.

**Review by ocefpaf — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jGaboardi) [reply README.md]: LOL. OK. I should have know that...

Comment (jGaboardi) [reply README.md]: Does this mean I should increase the build number in `meta.yml`?

Comment (ocefpaf) [reply README.md]: No need, this is more for the infrastructure, it will give you commit rights.

**Comment by ocefpaf** [comment]
@conda-forge-admin please rerender

**Review by ocefpaf — APPROVED** [review]

**Pull request merged**